aspnet/mvc/views/forms/sample/final/ViewModels/AddressViewModel.cs

```diff
@@ -0,0 +1,17 @@
+﻿
+namespace FormsTH.ViewModels
+{
+    public class AddressViewModel
+    {
+        public string AddressLine1 { get; set; }
+    }
+
+    public class RegisterViewModel2
+    {
+        public string Email { get; set; }
+
+        public string Password { get; set; }
```

Comment:
@dougbu  but then I'd have to introduce the input TH too - and that's the next section. The HTML with `` type="password"`` is exactly the same as the HTML generated by the TH when the TH add  ``type="password"`` - but you make a good point. I'll call out in the doc the security advantage the Input TH has over pure HTML - `` type="password"`` is generated for you in every razor view where you use that model with a input TH.

And yes I did check the responses